src/leo-server-app/src/routes/satellite.ts

```diff
@@ -1,16 +1,50 @@
+import * as dotenv from "dotenv";
+import { TLEResponse } from "../types/satellites";
+
+dotenv.config({ path: `.env.local`, override: true });
+
 const express = require("express");
+var spacetrack = require("spacetrack");
```

Comment:
It's kinda a nitpick but could you change all the 'var' declarations to 'let', 'let' is the newer convention of declaring variables